__tests__/app.test.js

```diff
@@ -207,5 +206,72 @@ describe("APP", () => {
           });
       });
     });
+    describe('7/POST/api/articles/articleid/comment', () => {
+      const commentToAdd = {
+        username: 'butter_bridge',
+        body: 'This article is amazing.  This has changed my life' 
+      }
+      test('responds with an object with the posted comment / correct properties', () => {
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send(commentToAdd)
+        .expect(201)
+        .then(({body}) => {
+          expect(Object.keys(body.commentAdded)).toHaveLength(6)
+          expect(body.commentAdded).toHaveProperty("comment_id");
+          expect(body.commentAdded).toHaveProperty("article_id");
+          expect(body.commentAdded).toHaveProperty("body");
+          expect(body.commentAdded).toHaveProperty("author");
+          expect(body.commentAdded).toHaveProperty("created_at");
+          expect(body.commentAdded).toHaveProperty("votes")
+        })
```

Comment:
it would be nice to also test the value of these keys too, to make sure that we are getting back the data we want